hrisbenefit.go

```diff
@@ -125,6 +125,7 @@ type BenefitFrequency string
 const (
 	BenefitFrequencyOneTime       BenefitFrequency = "one_time"
 	BenefitFrequencyEveryPaycheck BenefitFrequency = "every_paycheck"
+	BenefitFrequencyMonthly       BenefitFrequency = "monthly"
```

Comment:
This is good but let's exclude this for now as per https://github.com/Finch-API/finch-api-go/pull/21/commits/723b3f7bc37f6058603c475656b9f086154874ff#r1344028515